server/repository/assessment/createAssessment.ts

```diff
@@ -0,0 +1,19 @@
+import { Assessment } from '@core/meta/assessment'
+import { BaseProtocol, DB } from '@server/db'
+import { Objects } from '@core/utils'
+
+export const createAssessment = async (
+  params: {
+    assessment: Assessment
+  },
+  client: BaseProtocol = DB
+): Promise<Assessment> => {
+  const { assessment } = params
+
+  const query = `
+  insert into assessment (props) values ('${JSON.stringify(assessment.props)}'::jsonb) returning  *;
+
+`
+  const ret = await client.query<Assessment>(query)
```

Comment:
you could even call client.one<Assessment>

Reply:
you could even return ```client.one<Assessment>``` and pass camelize as third argument if you want to shorten the code